### PR TITLE
Add IPVanish Latest

### PR DESCRIPTION
**Pull request by olanmatt**

### CODE REVIEW

File: Casks/ipvanish.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'ipvanish' do
```
Comment (vitorgalvao) [inline Casks/ipvanish.rb]: Needs to be `ipvanish-vpn`, as per the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_token_reference.md).

File: Casks/ipvanish.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  version :latest
+  sha256 :no_check
+
+  url 'https://www.ipvanish.com/software/osx/IPVanish.dmg'
+  name 'IPVanish'
+  homepage 'https://www.ipvanish.com/'
+  license :closed
```
Comment (vitorgalvao) [inline Casks/ipvanish.rb]: Is the app itself paid? It seems like the app might be free, which makes this `:gratis`.

**Comment by olanmatt** [comment]
Fixed mistakes noted by @vitorgalvao in 6bcabfb.

**Comment by vitorgalvao** [comment]
Looks good, could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by olanmatt** [comment]
@vitorgalvao, done!

**Pull request merged**